raincatcher-generation-utils/README.md

```diff
@@ -0,0 +1,65 @@
+# Raincatcher Generation Utils
+
+This is a utility tool for generating users, workorders and csv data for running
+raincatcher performance tests against. Currently settings are for running tests
+locally.
+
+## Getting Started
+
+Fist install the depenedencies of the tools with:
+
+```
+npm install
+```
+
+### Generating Users
+
+Run the following command to generate the associated user files for the wfm
+demo, whilst changing `numberOfUsers` to the number of users you want to 
+produce.
+
+```
+npm run generate-users -- ./output/ numberOfUsers
+```
+
+This will output two important files in the `passport-users` and `wfm-users`
+folders in the `output` folder. Overwrite
+`raincatcher-core/demo/server/src/modules/passport-auth/users.json` and 
+`raincatcher-core/demo/server/src/modules/wfm-user/users.json` with the produced
+files, and when the server is started the new users will be able to authenticate
+into the system.
+
+A `user_ids.json` an a `users.csv` is also created. The `users.csv` file can be 
```

Comment:
This may be limited to 300 users on real tests.
We should not reuse the same user as this will cause collisions.